pages/docs/tooling/tooling-builtin.md

```diff
@@ -69,23 +71,47 @@ binprecice check FILE [ PARTICIPANT [ COMMSIZE ] ]
 
 The `check` runs the preCICE configuration parsing and checking logic on the given configuration file.
 This will find the majority of the configuration mistakes without having to start a simulation.
+These checks include wrong tags and attribute values, and more elaborate naming checks.
+More intricate logic, such as checks if all necessary data are exchanged in a Coupling Schemes, are not covered.
```

Comment:
```suggestion
More advanced logic, such as checks if all necessary data are exchanged in a coupling scheme, are not covered.
```

(we should use rather simpler vocabulary in documentation, as the goal is clarity and the target audience mostly non-native speakers)